<!DOCTYPE html>
<html lang="en" dir="auto">

<head><meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>Structural and Molecular Biology | Yatharth Bhasin</title>
<meta name="keywords" content="classnotes">
<meta name="description" content="IBB Lecture class notes.">
<meta name="author" content="Yatharth Bhasin">
<link rel="canonical" href="https://yatharthb97.github.io/classnotes/week4/">
<meta name="google-site-verification" content="XYZabc">
<meta name="yandex-verification" content="XYZabc">
<meta name="msvalidate.01" content="XYZabc">
<link crossorigin="anonymous" href="/assets/css/stylesheet.969e008100311592035bfe287f78f5312356e1927b1b0aab5f3c62868852b71f.css" integrity="sha256-lp4AgQAxFZIDW/4of3j1MSNW4ZJ7GwqrXzxihohStx8=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://yatharthb97.github.io/images/website_tile.svg">
<link rel="icon" type="image/png" sizes="16x16" href="https://yatharthb97.github.io/%3Clink%20/%20abs%20url%3E">
<link rel="icon" type="image/png" sizes="32x32" href="https://yatharthb97.github.io/%3Clink%20/%20abs%20url%3E">
<link rel="apple-touch-icon" href="https://yatharthb97.github.io/%3Clink%20/%20abs%20url%3E">
<link rel="mask-icon" href="https://yatharthb97.github.io/%3Clink%20/%20abs%20url%3E">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript>




<!DOCTYPE html>
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.16.0/dist/katex.min.css" integrity="sha384-Xi8rHCmBmhbuyyhbI88391ZKP2dmfnOl4rT9ZfRI7mLTdk1wblIUnrIq35nqwEvC" crossorigin="anonymous">
<script defer src="https://cdn.jsdelivr.net/npm/katex@0.16.0/dist/katex.min.js" integrity="sha384-X/XCfMm41VSsqRNQgDerQczD69XqmjOOOwYQvr/uuC+j4OPoNhVgjdGFwhvN02Ja" crossorigin="anonymous"></script>
<script defer src="https://cdn.jsdelivr.net/npm/katex@0.16.0/dist/contrib/auto-render.min.js" integrity="sha384-+XBljXPPiv+OzfbB3cVmLHf4hdUFHlWNZN5spNQ7rmHTXpd7WvJum6fIACpNNfIR" crossorigin="anonymous"></script>
<script>
    document.addEventListener("DOMContentLoaded", function() {
        renderMathInElement(document.body, {
          
          
          delimiters: [
              {left: '$$', right: '$$', display: true},
              {left: '$', right: '$', display: false},
          ],
          
          throwOnError : true
        });
    });
</script>



<script type="application/javascript">
var doNotTrack = false;
if (!doNotTrack) {
	(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
	(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
	m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
	})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
	ga('create', 'UA-178533000-1', 'auto');
	
	ga('send', 'pageview');
}
</script><meta property="og:title" content="Structural and Molecular Biology" />
<meta property="og:description" content="IBB Lecture class notes." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://yatharthb97.github.io/classnotes/week4/" />
<meta property="og:image" content="https://yatharthb97.github.io/%3Cimage%20path/url%3E" /><meta property="article:section" content="classnotes" />
<meta property="article:published_time" content="2022-09-26T09:48:53&#43;01:00" />
<meta property="article:modified_time" content="2022-09-26T09:48:53&#43;01:00" /><meta property="og:site_name" content="Yatharth Bhasin" />

<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://yatharthb97.github.io/%3Cimage%20path/url%3E" />
<meta name="twitter:title" content="Structural and Molecular Biology"/>
<meta name="twitter:description" content="IBB Lecture class notes."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [
    {
      "@type": "ListItem",
      "position":  1 ,
      "name": "Class Notes üìö",
      "item": "https://yatharthb97.github.io/classnotes/"
    }, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Structural and Molecular Biology",
      "item": "https://yatharthb97.github.io/classnotes/week4/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "Structural and Molecular Biology",
  "name": "Structural and Molecular Biology",
  "description": "IBB Lecture class notes.",
  "keywords": [
    "classnotes"
  ],
  "articleBody": "Principles of Protein Structures by Casper Goverde Group research: Design of protein molecules for novel functional and behavioural activities. Also study proteins using variety of techniques like ML and computational design. Moplecular Interactions in Biomolecules Non-covalent interactions modelled via Lennard-Jones, consists of van der Waals interactions, hydrogen bonds. Two neutral atoms in close proximity get polarised ! Van der waals radius is r1 + r2 and is the point where the repulsive interactions become dominent. Hydrogen bonds can be thought of as a dipole-dipole interaction but on a molecular scale. Iconic interactions are the strongest, however, water ions severly reduce electrostatic interaction. In vacuum its ~50 kJ/mol, whereas in water its ~6 kJ/mol. Moleculaes of Life Nucleic Acids Proteins : Made of Peptides Lipids : Fats Glycans: Sugars flowchart LR Genetic-Material --\u003e Primary-Structure --\u003e Secondary-Structure --\u003e Function((Function)) --\u003e Tertiary-Structure --\u003e Quartarnary-Structure Proteins have an amino group, a carboxyl group, and 20 different kinds of R groups.\nThey are built out of 20 amino acids encoded in DNA.\nThe amino acids can be classified based on the type of Carboxyl group.\nPolar and non-polar amino acids. Polar amino acids can form hydrogen bonds with itself and other amino acids. They are also hydrophobic. Negatively and positively charged amino acids. Their sidechains are charged at pH=7. Synthesis of protein: condensation reaction where amino acid of one ‚Ä¶TODO\nThe sequence of a protein is written from the N-terminus (one of the free $NH_3$) to the C-terminius (the one with $COO^-$).\nbackbobne + sidechain + actual witten structure + carboxyl extension.\nHydrophobicity of amino acids is an important feature.\nResidue: the central kernal of the proteins. Find out why a Residue is called a residue. TODO\nSome amino acids are special:\nCysteine can form disulphide bridges.\nflowchart LR Cysteine -.\"+\".- Cysteine2(Cysteine) --Oxidation--\u003e Cystine Cystine --Reduction--\u003e Cysteine2 Glycine doesn‚Äôt have a sidechain group.\nProline‚Äôs sidechain group is covalently bound to the nitrogen of the peptide bond.\nProtein Secondary Structures $\\alpha$-helix form to stabilize the hydrophobic core. It forms by making H-bonds between Hydrogen and oxygen.\n$\\beta$-sheets form hydrogen bonds with its whole backbone.\nUsually the alpha and beta sheets are sometimes misaligned and imperfect.\nAlpha-helix\nIn the alpha-helix, the carbonyl oxygen of residue ‚Äúi‚Äù forms a hydrogen bond with the amide of the residue ‚Äúi+4‚Äù. There are only right handed alpha-helixes, except for a handful of exceptions. Beta-sheets\nCan be wither parallel and antiparallel. They have roughly the same binding energy. In a beta-sheet, carbonyl oxygen and amides form hydrogen bonds between the strands, i.e between amino acids far away from each other in primary sequence. Proteins can be composed of one of the two secondary structures or be mixed.\nBeyond Secondary Structures The tertiary structure, which is the three-dimensional organisation of the secondary structure elements. It is also referred to as the protein fold. Quaternary structures refers to the association of different polypeptide chains (subunits) into a multinumeric complex. Protein domains are fundamental units of tertiary structue. it forms an independnt structural domin. They are often units of function. Domains have the same fold yet different amino acid sequences. Protein domains are very stable to mutations. ANd they have been selected via evolution for the very specific reason. The stability of the folded structure results primarily from the hydrophobic regions folding in together. Two proteins with different sequences can have the same sequence. Conformations and Folding Protein conformational changes does not require breaking bonds. The peptide bond has partial double bond character and therefore are sterically hindered. Protein folding induces conformational changes in the backbone. The backbone torsion angles $\\phi$ and $\\psi$ determine that conformation of the protein chain. The Ramachandran diagram define the restrictions on backbone conformation. $\\psi$ : TODO $\\phi$ : TODO L: Left handed helix You see a lot of Glycines in loops as they are flexible because of the absence of any side chains. Prolines are in Cis conformation so they ususally ‚Äúkink‚Äù the alpha-helix. Not generally found anywhere else. Secondary structural elements are conected to form simple motifs. Given motifs, you can create many folds. The interactions between the atoms in a protein control the folding of the protein into a well defined structure (native structure): Thermodynamic hyposthesis of protein folding. It was demonstrated by the famous Anfinsen Experiment which opened the folded structure of a protein and allowed it to relax in a controlled manner to recover the folded structure, This information is encoded in their amino acid sequence. Evolution and Protein Structure by Casper Goverde Reading Suggestion: The Molecules of Life (Chapter 5)\nProtein structures are conserved during evolution while amino acids vary.\nMotifs that are crutial for protein folding are conserved during evolution.\nYou can use protein globins to classify species at the molecular level.\nDue to different selection pressures amd line seperation, protein sequences diverge between species.\nHow to quantify differences in sequences? Use a substitution matrix (BLOSUM). We use a matrix of probailities. Each entry in the matrix is called a substitution score ($S_{ij}$). The higher the score (positive), the more likely is the substition.\nThe BLOSUM substitution score is the logarithm of the likelihood that one aa in a protein is replaced by another during evolution.\n$S_{ij}$ is related to the frequency of substititon over random chance.\nBLAST (Basic Local Alignmnet Search Tool) : Identify novel proteins, find functional relationships and sequence motifs and patterns.\nTOP-7 is the first synthetic motif.\nThe most used measure is the Root Mean Square Deviation (RMSD).\nDomains are ‚Äúself contained units‚Äù that can be placed in different proteins.\nPyMol Excercises Superimposition is done by fitting the polypeptide backbone of your target protein on the reference protein. Best fit will have smallest Rms (root mean square) value.\nWhat is PSIPRED?\nProtein crystals have a high solvent content, which you can also observe as large spaces between the proteins in the Electron density maps.\nDay 2\nX-ray Crystallography The further out spots the higher the resolution of the protein structure.\nThe dark ring ~3√Ö is caused by water molecules un the crystal.\nflowchart LR crystals --\"x-rays\"--\u003e diffraction-pattern --\u003e electron-density-maps--\u003eatomic-structure atomic-structure --refinement--\u003ediffraction-pattern Methods to check crystallization conditions for proteins: Hanging Drop- and Sitting Drop- Vapour Diffusion.\nTypical concentration for X-ray diffraction: TODO\nHigh B-factor: low confidence in position of atom, which also corresponds to low electron densities.\nIn 2020 the number of structures in PDB reached 171,916.\nNuclear Magnetic Resonance (NMR) Resonance Frequency in NMR: $$ R_f = \\frac{2 \\mu_BB_o}{h} $$\nResonance frequency depends on nature of the nucleus and its chemical environment.\nResonance frequency is directly proportional to the strength of the external applied magnetic field.\nWe commonly use $H^1$ and $C^{13}$ for labelling as they are NMR active.\nNuclear Overhauser Effect (NOE): Provides information on spatial proximity of protons in protein. Protons that are close by influence each others chemical shifts.\nCrystallography si also applicable to very big proteins and complexes.\nNMR: Dynamic ensemble of several energy-minimized structures, whereas, X-ray crystallography gives a static picture of protein.\nNMR has a size limit: \u003e25kDa is very difficult.\nCryo-EM Cryo-EM prevents electron degradation of proteins.\nAbbe‚Äôs Relationship: $d_R = 1.22 \\lambda /NA$. Working out this formula gives us a $d_R$ to be 1.4√Ö.\nflowchart LR biochemical-preperation --\u003e cryo-em-sample-preparation --\u003e Imaging --\u003e data-collection(data-collection) dc(data-collection)--\u003eimage-processing--\u003ereconstruction--\u003e structural-analysis--\u003e Model Generally Liquid Ethane is used.\n3D images are reconstructed from 2D segment images from all angles.\nCryo-EM can do very big protein structures.\nBioinformatics and protein structure Uniprot : Entries are manually annotated. This is the first stop to check an unknown protein. It contains references to other databases as well. There are multiple .pdb files for a single protein corresponsiong to all the different experiments done with the protein. They might have different resolutions, and precision. PhosphoSite Plus : provides comprehensive information and tools for the study of protein post-translational modifications (PTMs) including phosphorylation, acetylation, and more. STRING: All about protein interations. It contains protein-protein interaction data. PDB: The Protein Data Bank. Drugbank: Database of drugs. ChEMBL: Drug database where searches can also be compound-centric using different formats. AlphaFold flowchart LR Sequence --\u003e Structure --\u003e Function Each step involves shrinking of the total phase space of that particular function.\nCritical Assessment of protein Structure Prediction (CASP) - competition held for prediction of Protein Structure. The FOM is GDT (Global Distance Test). AlphaFold 1 was the first to combine Multiple Sequence Alignment (MSA) and Deep Neural Networks. MSA finds similar sequence across different species, aligns, and then find ‚Äúsuitable mutations‚Äù or the set of all possible mutations (coevolution). If something mutates to a more positive base, the other base can mutate to something more negative, but not positive. We calculate distance matrix -\u003e Correlation matrix. A neural network can approximate any function, given that your neural network has arbitrary width and we have an infinite dataset. AlphaFold also uses a Convolutional neural networks to recognise patterns. Output is the distance matrix, and the distance matrix of the two dihedral angles $\\phi$ and $\\psi$. The Input for AlphaFold-2 is the MSA. AlphaFold-1:\ngraph LR 1(Sequence and MSA features)--\u003e2(Deep Neural Networks)--\u003e3(Distance and Torosional distribution predictions) AlphaFold-2:\nBad templates are inserted so AF won‚Äôt copy template features.\nAMBER relaxation optimizes the final structure.\n‚ÄúSelf-distillation training‚Äù - Using your own predictions as a dataset.\nHuman vs Other Species -\u003e Prediction Score Comparision\nIs all AlphaFold code open source? : Everything is opensource.\nAutocompletion is based on the Attention Matrix, whcih comes from a language model.\nAF also does a bunch of other things: protein-protein interactions, docking, predicting binding sites, etc.\nColabFold is a better version of AlphaFold 2. It is a lot faster with improved MSA step. It also allows for complex predictions.\nAlphaFold 2 practice notebook\nDay 3\nBinding Spontaneous process at constant temperature and pressure always involves decrease in the Gibbs Free energy of the system ($dG \u003c 0$).\nIt accounts for both enthalpic and entropic contributions of the system. $$ dG = dH - TdS $$\nEquilibrium Constants:\n$$ \\Delta G^0 = -RTlnK_{eq} \\rightarrow K_{eq} = e^{- \\frac {\\Delta G^0}{RT}} $$\nHomo Sapiens: ~15, 000 proteins expresses. 135,000 pp-interactions. Specificity is the ratio of binding affinities. They are theoreticlly not related. Innature, Higher affinity implies lower specificity. Binding sites usually have a hydrophobic core and then on the rim, you have bases that have specificity. Whwn fraction $f$ is half in a reaction, out dissociation constant $K_D$ equals the ligand concentration $L$. Univeral binding isotherm is expressed in terms of $K_D$. Drug Binding by Proteins Find a Lead compound. That bind to the receptor. They are usually not very specific and affinative. Iterate the lead compound to imrove specificity and affinity. Competative inhibitors: $IC_{50}$ is the concentration of the inhibitor that reduces the binding of the protein by half maximal value. $IC_{50}s$ are used rather than a single value because the activity of the drug may vary between the population just because of random mutations. Energetics of the Binding: There is entropic loss but enthalpic gain. $\\Delta G = \\Delta H - T \\Delta S$ The affinity for a protein for a ligand is characterized by the dissociation constant, $K_D$. The strength of binding interactions is often correlated with hydrophobicity. MASIF: Surface-Based Protein-Protein Interaction Prediction All informtion required for predicting binding is present on the surface. Exploring the potential of deep learning in de novo protein design Protein perform a variety of functions\nSignalling Immunity Structure Catalysis Transport Other functions (adapted for use):\nTherapeutics Biosensors Enzymes Biomaterials Proteins as domains:\ngraph LR Sequence --\u003e Structure --\u003e Function As a protein enginner you reverse the graph in your approach.\nMotif grafting: Take parts of proteins (motifs) and combine them at the sequence level.\nClassic de novo protein design is limited by already known structures (the PDB database). The top-down design method\nA better approach is the de novo protein design: It takes a Bottom-up functional protein design. It is much more computationally expensive and suffers from a very low success rate.\nAdded an MC Step to the protein iteration step for scoring random mutations.\n‚ÄúOracle models‚Äù\nExercise Notes ‚ÄúMaSIF receives a protein surface as an input and outputs a predicted score for the surface on the likelihood of being involved in a PPI.‚Äù\n‚Äî Assignment sheet\nMaSIF can be used for iteratively for negative designing i.e. to remove features from the protein structure.\nDay 4\nDNA structure and replication Marco Fumasoni The famous Photo 51 which was an X-ray crystallographic image of the DNA double helix strand. DNA replication proceeds only in one direction. It is because of the presence of phosphtes on the one side only, which is required for catalysis. The replication movement creates a ‚Äúreplication fork‚Äù, which leads to a leading strand (continuous synthesis) and lagging strand (discontinuous sequence). The fragments on the lagging strand are vcalled Okazaki fragments. Replisome; Average human chromosome: 150 Mbp Average Eukaryotic replisome speed: ~50 nt/s It will take 400 hours for the replication to occur. Hence there are 100s of origins for replication on a chromosome. How to identify DNA replication origins? (which are parts ) : Chop DNA, make some kind of loops with URA3. Only segments with Origins replicate and survive to grow into colonies. Deep Sequencing: sequencing of DNA at different time points gives rise to a frequency distribution, where the modes give you the Origin points. This method can also give you the speed of the fork , etc. With Cloning we can find all potential Origin sites, however, with Deep Sequencing, you find the Origins that are being actually used by the cell. ",
  "wordCount" : "2250",
  "inLanguage": "en",
  "image":"https://yatharthb97.github.io/%3Cimage%20path/url%3E","datePublished": "2022-09-26T09:48:53+01:00",
  "dateModified": "2022-09-26T09:48:53+01:00",
  "author":{
    "@type": "Person",
    "name": "Yatharth Bhasin"
  },
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://yatharthb97.github.io/classnotes/week4/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "Yatharth Bhasin",
    "logo": {
      "@type": "ImageObject",
      "url": "https://yatharthb97.github.io/images/website_tile.svg"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://yatharthb97.github.io/" accesskey="h" title="Yatharth Bhasin (Alt + H)">
                <img src="https://yatharthb97.github.io/images/header_button.gif" alt="logo" aria-label="logo"
                    height="30">Yatharth Bhasin</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
            <li>
                <a href="https://yatharthb97.github.io/aboutme/" title="about me">
                    <span>about me</span>
                </a>
            </li>
            <li>
                <a href="https://yatharthb97.github.io/projects/" title="projects">
                    <span>projects</span>
                </a>
            </li>
            <li>
                <a href="https://yatharthb97.github.io/blog/" title="blog">
                    <span>blog</span>
                </a>
            </li>
            <li>
                <a href="https://yatharthb97.github.io/publications/" title="publications">
                    <span>publications</span>
                </a>
            </li>
            <li>
                <a href="https://yatharthb97.github.io/search/" title="search (Alt &#43; /)" accesskey=/>
                    <span>search</span>
                </a>
            </li>
            <li>
                <a href="https://yatharthb97.github.io/gallery/" title="gallery">
                    <span>gallery</span>
                </a>
            </li>
        </ul>
    </nav>
</header>
<main class="main">

<article class="post-single">
  <header class="post-header">
    <div class="breadcrumbs"><a href="https://yatharthb97.github.io/">Home</a>&nbsp;¬ª&nbsp;<a href="https://yatharthb97.github.io/classnotes/">Class Notes üìö</a></div>
    <h1 class="post-title">
      Structural and Molecular Biology
    </h1>
    <div class="post-description">
      IBB Lecture class notes.
    </div>
    <div class="post-meta"><span title='2022-09-26 09:48:53 +0100 +0100'>September 26, 2022</span>&nbsp;¬∑&nbsp;11 min&nbsp;¬∑&nbsp;2250 words&nbsp;¬∑&nbsp;Yatharth Bhasin&nbsp;|&nbsp;<a href="https://github.com/yatharthb97/yatharthb97.github.io/tree/main/content//classnotes/week4.md" rel="noopener noreferrer" target="_blank">Suggest Changes</a>

</div>
  </header> <div class="toc">
    <details >
        <summary accesskey="c" title="(Alt + C)">
            <span class="details">Table of Contents</span>
        </summary>

        <div class="inner"><ul>
                <li>
                    <a href="#principles-of-protein-structures" aria-label="Principles of Protein Structures">Principles of Protein Structures</a><ul>
                        
                <li>
                    <a href="#moplecular-interactions-in-biomolecules" aria-label="Moplecular Interactions in Biomolecules">Moplecular Interactions in Biomolecules</a></li>
                <li>
                    <a href="#moleculaes-of-life" aria-label="Moleculaes of Life">Moleculaes of Life</a></li>
                <li>
                    <a href="#protein-secondary-structures" aria-label="Protein Secondary Structures">Protein Secondary Structures</a></li>
                <li>
                    <a href="#beyond-secondary-structures" aria-label="Beyond Secondary Structures">Beyond Secondary Structures</a></li>
                <li>
                    <a href="#conformations-and-folding" aria-label="Conformations and Folding">Conformations and Folding</a></li></ul>
                </li>
                <li>
                    <a href="#evolution-and-protein-structure" aria-label="Evolution and Protein Structure">Evolution and Protein Structure</a><ul>
                        
                <li>
                    <a href="#how-to-quantify-differences-in-sequences" aria-label="How to quantify differences in sequences?">How to quantify differences in sequences?</a></li></ul>
                </li>
                <li>
                    <a href="#pymol-excercises" aria-label="PyMol Excercises">PyMol Excercises</a></li>
                <li>
                    <a href="#x-ray-crystallography" aria-label="X-ray Crystallography">X-ray Crystallography</a></li>
                <li>
                    <a href="#nuclear-magnetic-resonance-nmr" aria-label="Nuclear Magnetic Resonance (NMR)">Nuclear Magnetic Resonance (NMR)</a></li>
                <li>
                    <a href="#cryo-em" aria-label="Cryo-EM">Cryo-EM</a></li>
                <li>
                    <a href="#bioinformatics-and-protein-structure" aria-label="Bioinformatics and protein structure">Bioinformatics and protein structure</a></li>
                <li>
                    <a href="#alphafold" aria-label="AlphaFold">AlphaFold</a></li>
                <li>
                    <a href="#binding" aria-label="Binding">Binding</a><ul>
                        
                <li>
                    <a href="#drug-binding-by-proteins" aria-label="Drug Binding by Proteins">Drug Binding by Proteins</a></li></ul>
                </li>
                <li>
                    <a href="#exploring-the-potential-of-deep-learning-in-de-novo-protein-design" aria-label="Exploring the potential of deep learning in de novo protein design">Exploring the potential of deep learning in de novo protein design</a></li>
                <li>
                    <a href="#exercise-notes" aria-label="Exercise Notes">Exercise Notes</a></li>
                <li>
                    <a href="#dna-structure-and-replication" aria-label="DNA structure and replication">DNA structure and replication</a>
                </li>
            </ul>
        </div>
    </details>
</div>

  <div class="post-content"><h1 id="principles-of-protein-structures">Principles of Protein Structures<a hidden class="anchor" aria-hidden="true" href="#principles-of-protein-structures">#</a></h1>
<center> by Casper Goverde </center>
<ul>
<li>Group research: Design of protein molecules for novel functional and behavioural activities. Also study proteins using variety of techniques like ML and computational design.</li>
</ul>
<h2 id="moplecular-interactions-in-biomolecules">Moplecular Interactions in Biomolecules<a hidden class="anchor" aria-hidden="true" href="#moplecular-interactions-in-biomolecules">#</a></h2>
<ul>
<li>Non-covalent interactions modelled via Lennard-Jones, consists of <strong>van der Waals interactions, hydrogen bonds.</strong></li>
<li><strong>Two neutral atoms in close proximity get polarised !</strong></li>
<li>Van der waals radius is <strong>r1 + r2</strong> and is the point where the repulsive interactions become dominent.</li>
<li>Hydrogen bonds can be thought of as a dipole-dipole interaction but on a molecular scale.</li>
<li>Iconic interactions are the strongest, however, water ions severly reduce electrostatic interaction.</li>
<li>In vacuum its ~50 kJ/mol, whereas in water its ~6 kJ/mol.</li>
</ul>
<h2 id="moleculaes-of-life">Moleculaes of Life<a hidden class="anchor" aria-hidden="true" href="#moleculaes-of-life">#</a></h2>
<ol>
<li>Nucleic Acids</li>
<li>Proteins : Made of Peptides</li>
<li>Lipids : Fats</li>
<li>Glycans: Sugars</li>
</ol>
<div class="mermaid">flowchart LR
	Genetic-Material --> Primary-Structure --> Secondary-Structure --> Function((Function)) -->  Tertiary-Structure --> Quartarnary-Structure
</div>
<ul>
<li>
<p>Proteins have an amino group, a carboxyl group, and 20 different kinds of R groups.</p>
</li>
<li>
<p>They are built out of 20 amino acids encoded in DNA.</p>
</li>
<li>
<p>The amino acids can be classified based on the type of Carboxyl group.</p>
<ul>
<li><strong>Polar and non-polar amino acids</strong>. Polar amino acids can form hydrogen bonds with itself and other amino acids. They are also hydrophobic.</li>
<li><strong>Negatively and positively charged amino acids</strong>. Their sidechains are charged at pH=7.</li>
</ul>
</li>
<li>
<p>Synthesis of protein: condensation reaction where amino acid of one &hellip;TODO</p>
</li>
<li>
<p>The sequence of a protein is written from the N-terminus (one of the free $NH_3$)  to the C-terminius (the one with $COO^-$).</p>
</li>
<li>
<p>backbobne + sidechain + actual witten structure + carboxyl extension.</p>
</li>
<li>
<p>Hydrophobicity of amino acids is an important feature.</p>
</li>
<li>
<p>Residue: the central kernal of the proteins. <strong>Find out why a Residue is called a residue. TODO</strong></p>
</li>
<li>
<p>Some amino acids are special:</p>
<ul>
<li>
<p>Cysteine can form disulphide bridges.</p>
<div class="mermaid">flowchart LR
	Cysteine -."+".- Cysteine2(Cysteine) --Oxidation--> Cystine
	Cystine --Reduction--> Cysteine2
</div>
</li>
<li>
<p>Glycine doesn&rsquo;t have a sidechain group.</p>
</li>
<li>
<p>Proline&rsquo;s sidechain group is covalently bound to the nitrogen of the peptide bond.</p>
</li>
</ul>
</li>
</ul>
<h2 id="protein-secondary-structures">Protein Secondary Structures<a hidden class="anchor" aria-hidden="true" href="#protein-secondary-structures">#</a></h2>
<ul>
<li>
<p>$\alpha$-helix form to stabilize the hydrophobic core. It forms by making H-bonds between Hydrogen and oxygen.</p>
</li>
<li>
<p>$\beta$-sheets form hydrogen bonds with its whole backbone.</p>
</li>
<li>
<p>Usually the alpha and beta sheets are sometimes misaligned and imperfect.</p>
</li>
<li>
<p><strong>Alpha-helix</strong></p>
<ul>
<li>In the alpha-helix, the carbonyl oxygen of residue &ldquo;i&rdquo; forms a hydrogen bond with the amide of the residue &ldquo;i+4&rdquo;.</li>
<li>There are only right handed alpha-helixes, except for a handful of exceptions.</li>
</ul>
</li>
<li>
<p><strong>Beta-sheets</strong></p>
<ul>
<li>Can be wither parallel and antiparallel. They have roughly the same binding energy.</li>
<li>In a beta-sheet, carbonyl oxygen and amides form hydrogen bonds between the strands, i.e between amino acids  far away from each other in primary sequence.</li>
</ul>
</li>
<li>
<p>Proteins can be composed of one of the two secondary structures or be mixed.</p>
</li>
</ul>
<h2 id="beyond-secondary-structures">Beyond Secondary Structures<a hidden class="anchor" aria-hidden="true" href="#beyond-secondary-structures">#</a></h2>
<ul>
<li>The tertiary structure, which is the three-dimensional organisation of the secondary structure elements. It is also referred to as the <strong>protein fold</strong>.</li>
<li>Quaternary structures refers to the association of different polypeptide chains (subunits) into a multinumeric complex.</li>
<li>Protein domains are fundamental units of tertiary structue. it forms an independnt structural domin. They are often units of function.</li>
<li>Domains have the same fold yet different amino acid sequences.</li>
<li>Protein domains are very stable to mutations. ANd they have been selected via evolution for the very specific reason.</li>
<li>The stability of the folded structure results primarily from the hydrophobic regions folding in together.</li>
<li>Two proteins with different sequences can have the same sequence.</li>
</ul>
<h2 id="conformations-and-folding">Conformations and Folding<a hidden class="anchor" aria-hidden="true" href="#conformations-and-folding">#</a></h2>
<ul>
<li>Protein conformational changes does not require breaking bonds.</li>
<li>The peptide bond has partial double bond character and therefore are sterically hindered.</li>
<li>Protein folding induces conformational changes in the backbone. The backbone torsion angles $\phi$ and $\psi$ determine that conformation of the protein chain.</li>
<li>The Ramachandran diagram define the restrictions on backbone conformation.</li>
<li>$\psi$ : TODO</li>
<li>$\phi$ : TODO</li>
<li>L: Left handed helix</li>
<li>You see a lot of Glycines in loops as they are flexible because of the absence of any side chains.</li>
<li>Prolines are in Cis conformation so they ususally &ldquo;kink&rdquo; the alpha-helix. Not generally found anywhere else.</li>
<li>Secondary structural elements are conected to form simple motifs. Given motifs, you can create many folds.</li>
<li>The interactions between the atoms in a protein control the folding of the protein into a well defined structure (native structure): <strong>Thermodynamic hyposthesis of protein folding</strong>. It was demonstrated by the famous Anfinsen Experiment which opened the folded structure of a protein and allowed it to relax in a controlled manner to recover the folded structure,</li>
<li>This information is encoded in their amino acid sequence.</li>
</ul>
<hr>
<h1 id="evolution-and-protein-structure">Evolution and Protein Structure<a hidden class="anchor" aria-hidden="true" href="#evolution-and-protein-structure">#</a></h1>
<center> by Casper Goverde </center>
<ul>
<li>
<blockquote>
<p>Reading Suggestion: The Molecules of Life (Chapter 5)</p>
</blockquote>
</li>
<li>
<p>Protein structures are conserved during evolution while amino acids vary.</p>
</li>
<li>
<p>Motifs that are crutial for protein folding are conserved during evolution.</p>
</li>
<li>
<p>You can use protein globins to classify species at the molecular level.</p>
</li>
<li>
<p>Due to different selection pressures amd line seperation, protein sequences diverge between species.</p>
</li>
</ul>
<h2 id="how-to-quantify-differences-in-sequences">How to quantify differences in sequences?<a hidden class="anchor" aria-hidden="true" href="#how-to-quantify-differences-in-sequences">#</a></h2>
<ul>
<li>
<p>Use a substitution matrix (BLOSUM). We use a matrix of probailities. Each entry in the matrix is called a substitution score ($S_{ij}$). The higher the score (positive), the more likely is the substition.</p>
</li>
<li>
<p>The BLOSUM substitution score is the logarithm of the likelihood that one aa in a protein is replaced by another during evolution.</p>
</li>
<li>
<p>$S_{ij}$ is related to the frequency of substititon over random chance.</p>
</li>
<li>
<blockquote>
<p>BLAST (Basic Local Alignmnet Search Tool) : Identify novel proteins, find functional relationships and sequence motifs and patterns.</p>
</blockquote>
</li>
<li>
<p>TOP-7 is the first synthetic motif.</p>
</li>
<li>
<p>The most used measure is the <strong>Root Mean Square Deviation (RMSD).</strong></p>
</li>
<li>
<p>Domains are &ldquo;self contained units&rdquo; that can be placed in different proteins.</p>
</li>
</ul>
<h1 id="pymol-excercises">PyMol Excercises<a hidden class="anchor" aria-hidden="true" href="#pymol-excercises">#</a></h1>
<ul>
<li>
<p>Superimposition is done by fitting the polypeptide backbone of your target protein on the reference protein. Best fit will have smallest Rms (root mean square) value.</p>
</li>
<li>
<blockquote>
<p>What is PSIPRED?</p>
</blockquote>
</li>
<li>
<p>Protein crystals have a high solvent content, which you can also observe as large spaces between the proteins in the Electron density maps.</p>
</li>
</ul>
<hr>
<p>Day 2</p>
<hr>
<h1 id="x-ray-crystallography">X-ray Crystallography<a hidden class="anchor" aria-hidden="true" href="#x-ray-crystallography">#</a></h1>
<ul>
<li>
<p>The further out spots the higher the resolution of the protein structure.</p>
</li>
<li>
<p>The dark ring ~3√Ö is caused by water molecules un the crystal.</p>
</li>
<li>
<div class="mermaid">flowchart LR
	crystals --"x-rays"--> diffraction-pattern --> electron-density-maps-->atomic-structure
	atomic-structure --refinement-->diffraction-pattern
</div>
</li>
<li>
<p>Methods to check crystallization conditions for proteins: Hanging Drop- and Sitting Drop- Vapour Diffusion.</p>
</li>
<li>
<p>Typical concentration for X-ray diffraction: TODO</p>
</li>
<li>
<p><strong>High B-factor</strong>: low confidence in position of atom, which also corresponds to low electron densities.</p>
</li>
<li>
<p>In 2020 the number of structures in PDB reached <strong>171,916</strong>.</p>
</li>
</ul>
<h1 id="nuclear-magnetic-resonance-nmr">Nuclear Magnetic Resonance (NMR)<a hidden class="anchor" aria-hidden="true" href="#nuclear-magnetic-resonance-nmr">#</a></h1>
<ul>
<li>
<p>Resonance Frequency in NMR:
$$
R_f = \frac{2 \mu_BB_o}{h}
$$</p>
</li>
<li>
<p>Resonance frequency depends on nature of the nucleus and its chemical environment.</p>
</li>
<li>
<p>Resonance frequency is directly proportional to the strength of the external applied magnetic field.</p>
</li>
<li>
<p>We commonly use $H^1$ and $C^{13}$ for labelling as they are NMR active.</p>
</li>
<li>
<p>Nuclear Overhauser Effect (NOE): Provides information on <strong>spatial proximity of protons in protein.</strong> Protons that are close by influence each others chemical shifts.</p>
</li>
<li>
<p>Crystallography si also applicable to very big proteins and complexes.</p>
</li>
<li>
<p>NMR: Dynamic ensemble of several energy-minimized structures, whereas, X-ray crystallography gives a static picture of protein.</p>
</li>
<li>
<p>NMR has a size limit: &gt;25kDa is very difficult.</p>
</li>
</ul>
<h1 id="cryo-em">Cryo-EM<a hidden class="anchor" aria-hidden="true" href="#cryo-em">#</a></h1>
<ul>
<li>
<p>Cryo-EM prevents electron degradation of proteins.</p>
</li>
<li>
<p>Abbe&rsquo;s Relationship: $d_R = 1.22 \lambda /NA$. Working out this formula gives us a $d_R$ to be 1.4√Ö.</p>
<div class="mermaid">
flowchart LR
	biochemical-preperation --> cryo-em-sample-preparation --> Imaging --> data-collection(data-collection)
	dc(data-collection)-->image-processing-->reconstruction--> structural-analysis--> Model
</div>
</li>
<li>
<p>Generally Liquid Ethane is used.</p>
</li>
<li>
<p>3D images are reconstructed from 2D segment images from all angles.</p>
</li>
<li>
<p>Cryo-EM can do very big protein structures.</p>
</li>
</ul>
<h1 id="bioinformatics-and-protein-structure">Bioinformatics and protein structure<a hidden class="anchor" aria-hidden="true" href="#bioinformatics-and-protein-structure">#</a></h1>
<ul>
<li><code>Uniprot</code> : Entries are manually annotated. This is the first stop to check an unknown protein. It contains references to other databases as well.</li>
<li>There are multiple <code>.pdb</code> files for a single protein corresponsiong to all the different experiments done with the protein. They might have different resolutions, and precision.</li>
<li><code>PhosphoSite Plus</code> : provides comprehensive information and tools for the study of protein post-translational modifications (PTMs) including phosphorylation, acetylation, and more.</li>
<li><code>STRING</code>: All about protein interations. It contains protein-protein interaction data.</li>
<li><code>PDB</code>: The Protein Data Bank.</li>
<li><code>Drugbank</code>:  Database of drugs.</li>
<li><code>ChEMBL</code>: Drug database where searches can also be compound-centric using different formats.</li>
</ul>
<h1 id="alphafold">AlphaFold<a hidden class="anchor" aria-hidden="true" href="#alphafold">#</a></h1>
<div class="mermaid">flowchart LR
	Sequence --> Structure --> Function
</div>
<p>Each step involves shrinking of the total phase space of that particular function.</p>
<ul>
<li>Critical Assessment of protein Structure Prediction (CASP) - competition held for prediction of Protein Structure. The FOM is GDT (Global Distance Test).</li>
<li>AlphaFold 1 was the first to combine Multiple Sequence Alignment (MSA) and Deep Neural Networks.</li>
<li>MSA finds similar sequence across different species, aligns, and then find &ldquo;suitable mutations&rdquo; or the set of all possible mutations (coevolution).
<ul>
<li>If something mutates to a more positive base, the other base can mutate to something more negative, but not positive.</li>
<li>We calculate distance matrix -&gt; Correlation matrix.</li>
</ul>
</li>
<li>A neural network can approximate any function, given that your neural network has arbitrary width and we have an infinite dataset.</li>
<li>AlphaFold also uses a Convolutional neural networks to recognise patterns.</li>
<li>Output is the distance matrix, and the distance matrix of the two dihedral angles $\phi$  and $\psi$.</li>
<li>The Input for AlphaFold-2 is the MSA.</li>
</ul>
<p><strong>AlphaFold-1:</strong></p>
<div class="mermaid">graph LR
	1(Sequence and MSA features)-->2(Deep Neural Networks)-->3(Distance and Torosional distribution predictions)
</div>
<p><strong>AlphaFold-2:</strong></p>
<p><img loading="lazy" src="/Users/byatharth/code/yatharthb97.github.io/static/images/classnotes/alphafold.jpeg" alt="Alphafold PipeLine (credits Casper)"  />
</p>
<ul>
<li>
<p>Bad templates are inserted so AF won&rsquo;t copy template features.</p>
</li>
<li>
<p>AMBER relaxation optimizes the final structure.</p>
</li>
<li>
<p>&ldquo;Self-distillation training&rdquo; - Using your own predictions as a dataset.</p>
</li>
<li>
<blockquote>
<p>Human vs Other Species -&gt; Prediction Score Comparision</p>
</blockquote>
</li>
<li>
<p><strong>Is all AlphaFold code open source?</strong> :  Everything is opensource.</p>
</li>
<li>
<p>Autocompletion is based on the Attention Matrix, whcih comes from a language model.</p>
</li>
<li>
<p>AF also does a bunch of other things: protein-protein interactions, docking, predicting binding sites, etc.</p>
</li>
<li>
<p>ColabFold is a better version of AlphaFold 2. It is a lot faster with improved MSA step. It also allows for complex predictions.</p>
</li>
<li>
<p><a href="https://colab.research.google.com/github/sokrypton/ColabFold/blob/main/AlphaFold2.ipynb#scrollTo=ADDuaolKmjGW">AlphaFold 2 practice notebook</a></p>
</li>
</ul>
<hr>
<p>Day 3</p>
<hr>
<h1 id="binding">Binding<a hidden class="anchor" aria-hidden="true" href="#binding">#</a></h1>
<ul>
<li>
<p>Spontaneous process at constant temperature and pressure always involves decrease in the Gibbs Free energy of the system ($dG &lt; 0$).</p>
</li>
<li>
<p>It accounts for both enthalpic and entropic contributions of the system.
$$
dG = dH - TdS
$$</p>
</li>
<li>
<p>Equilibrium Constants:</p>
</li>
</ul>
<p>$$
\Delta G^0 = -RTlnK_{eq} \rightarrow K_{eq} = e^{- \frac {\Delta G^0}{RT}}
$$</p>
<ul>
<li><strong>Homo Sapiens</strong>:  ~15, 000 proteins expresses. 135,000 pp-interactions.</li>
<li>Specificity is the ratio of binding affinities. They are theoreticlly not related. Innature, Higher affinity implies lower specificity.</li>
<li>Binding sites usually have a hydrophobic core and then on the rim, you have bases that have specificity.</li>
<li>Whwn fraction $f$ is half in a reaction, out dissociation constant $K_D$ equals the ligand concentration $L$.</li>
<li>Univeral binding isotherm is expressed in terms of $K_D$.</li>
</ul>
<h2 id="drug-binding-by-proteins">Drug Binding by Proteins<a hidden class="anchor" aria-hidden="true" href="#drug-binding-by-proteins">#</a></h2>
<ol>
<li>Find a <strong>Lead compound</strong>. That bind to the receptor. They are usually not very specific and affinative.</li>
<li>Iterate the lead compound to imrove specificity and affinity.</li>
</ol>
<ul>
<li>Competative inhibitors: $IC_{50}$ is the concentration of the inhibitor that reduces the binding of the protein by half maximal value.</li>
<li>$IC_{50}s$ are used rather than a single value because the activity of the drug may vary between the population just because of random mutations.</li>
<li>Energetics of the Binding: There is entropic loss but enthalpic gain. $\Delta G = \Delta H - T \Delta S$</li>
<li>The affinity for a protein for a ligand is characterized by the dissociation constant, $K_D$.</li>
<li>The strength of binding interactions is often correlated with hydrophobicity.</li>
<li>MASIF: Surface-Based Protein-Protein Interaction Prediction
<ul>
<li>All informtion required for predicting binding is present on the surface.</li>
</ul>
</li>
</ul>
<h1 id="exploring-the-potential-of-deep-learning-in-de-novo-protein-design">Exploring the potential of deep learning in de novo protein design<a hidden class="anchor" aria-hidden="true" href="#exploring-the-potential-of-deep-learning-in-de-novo-protein-design">#</a></h1>
<ul>
<li>
<p>Protein perform a variety of functions</p>
<ul>
<li>Signalling</li>
<li>Immunity</li>
<li>Structure</li>
<li>Catalysis</li>
<li>Transport</li>
</ul>
</li>
<li>
<p>Other functions (adapted for use):</p>
<ul>
<li>Therapeutics</li>
<li>Biosensors</li>
<li>Enzymes</li>
<li>Biomaterials</li>
</ul>
</li>
<li>
<p>Proteins as domains:</p>
<div class="mermaid">graph LR
	Sequence --> Structure --> Function
</div>
<p>As a protein enginner you reverse the graph in your approach.</p>
</li>
<li>
<p>Motif grafting: Take parts of proteins (motifs) and combine them at the sequence level.</p>
</li>
<li>
<p>Classic de novo protein design is limited by already known structures (the PDB database). The <strong>top-down design method</strong></p>
</li>
<li>
<p>A better approach is the <strong>de novo protein design</strong>: It takes a Bottom-up functional protein design. It is much more computationally expensive and suffers from a very low success rate.</p>
</li>
<li>
<p>Added an MC Step to the protein iteration step for scoring random mutations.</p>
</li>
<li>
<blockquote>
<blockquote>
<blockquote>
<p>&ldquo;Oracle models&rdquo;</p>
</blockquote>
</blockquote>
</blockquote>
</li>
</ul>
<h1 id="exercise-notes">Exercise Notes<a hidden class="anchor" aria-hidden="true" href="#exercise-notes">#</a></h1>
<ul>
<li>
<blockquote>
<p>‚ÄúMaSIF receives a protein surface as an input and outputs a predicted score for the surface on the likelihood of being involved in a PPI.‚Äù</p>
<p>‚Äî Assignment sheet</p>
</blockquote>
</li>
<li>
<p>MaSIF can be used for iteratively for negative designing i.e. to remove features from the protein structure.</p>
</li>
</ul>
<hr>
<p>Day 4</p>
<hr>
<h1 id="dna-structure-and-replication">DNA structure and replication<a hidden class="anchor" aria-hidden="true" href="#dna-structure-and-replication">#</a></h1>
<center> Marco Fumasoni</center>
<ul>
<li>The famous <strong>Photo 51</strong> which was an X-ray crystallographic image of the DNA double helix strand.</li>
<li>DNA replication proceeds only in one direction. It is because of the presence of phosphtes on the one side only, which is required for catalysis.</li>
<li>The replication movement creates a &ldquo;replication fork&rdquo;, which leads to a <strong>leading strand (continuous synthesis)</strong> and <strong>lagging strand (discontinuous sequence)</strong>.  The fragments on the lagging strand are vcalled <strong>Okazaki fragments</strong>.</li>
<li>Replisome;
<ul>
<li>Average human chromosome: 150 Mbp</li>
<li>Average Eukaryotic replisome speed: ~50 nt/s</li>
<li>It will take 400 hours for the replication to occur.</li>
<li>Hence there are 100s of origins for replication on a chromosome.</li>
</ul>
</li>
<li>How to identify DNA replication origins? (which are parts ) :
<ul>
<li>Chop DNA, make some kind of loops with URA3. Only segments with Origins replicate and survive to grow into colonies.</li>
</ul>
</li>
<li>Deep Sequencing: sequencing of DNA at different time points gives rise to a frequency distribution, where the modes give you the Origin points.
<ul>
<li>This method can also give you the speed of the fork , etc.</li>
</ul>
</li>
<li>With Cloning we can find all potential Origin sites, however, with Deep Sequencing, you find the Origins that are being actually used by the cell.</li>
<li></li>
</ul>


  </div>

  
   <script src="https://cdn.jsdelivr.net/npm/mermaid/dist/mermaid.min.js"></script>
   <script>
       let isDark = document.body.className.includes("dark")
       let mermaidTheme = (isDark) ? 'dark' : 'forest'
       let mermaidConfig = {
           theme: 'neutral',
           logLevel: 'fatal',
           securityLevel: 'strict',
           startOnLoad: true,
           arrowMarkerAbsolute: false
       };
       mermaid.initialize(mermaidConfig);
   </script>
  

  <footer class="post-footer">
    <ul class="post-tags">
      <li><a href="https://yatharthb97.github.io/tags/classnotes/">classnotes</a></li>
    </ul>

  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2022 <a href="https://yatharthb97.github.io/">Yatharth Bhasin</a></span>
    <span>
        Powered by
        <a href="https://gohugo.io/" rel="noopener noreferrer" target="_blank">Hugo</a> &
        <a href="https://github.com/adityatelange/hugo-PaperMod/" rel="noopener" target="_blank">PaperMod</a>
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
<script>
    document.querySelectorAll('pre > code').forEach((codeblock) => {
        const container = codeblock.parentNode.parentNode;

        const copybutton = document.createElement('button');
        copybutton.classList.add('copy-code');
        copybutton.innerHTML = 'copy';

        function copyingDone() {
            copybutton.innerHTML = 'copied!';
            setTimeout(() => {
                copybutton.innerHTML = 'copy';
            }, 2000);
        }

        copybutton.addEventListener('click', (cb) => {
            if ('clipboard' in navigator) {
                navigator.clipboard.writeText(codeblock.textContent);
                copyingDone();
                return;
            }

            const range = document.createRange();
            range.selectNodeContents(codeblock);
            const selection = window.getSelection();
            selection.removeAllRanges();
            selection.addRange(range);
            try {
                document.execCommand('copy');
                copyingDone();
            } catch (e) { };
            selection.removeRange(range);
        });

        if (container.classList.contains("highlight")) {
            container.appendChild(copybutton);
        } else if (container.parentNode.firstChild == container) {
            
        } else if (codeblock.parentNode.parentNode.parentNode.parentNode.parentNode.nodeName == "TABLE") {
            
            codeblock.parentNode.parentNode.parentNode.parentNode.parentNode.appendChild(copybutton);
        } else {
            
            codeblock.parentNode.appendChild(copybutton);
        }
    });
</script>
</body>

</html>
